lib/interop-csn.js

```diff
@@ -0,0 +1,35 @@
+
+function interopCSN(csn) {
+
+  for (let dn in csn.definitions) {
+    let def = csn.definitions[dn];
+    replaceAnnos(def);
+    if (def.kind === "entity")
```

Comment:
@stewsk @Fannon should we keep the "query" section in the csn interop?

Reply:
No, but the query section should already have been removed before in the call to "effectiveCsn" when a service name is provided as input parameter (https://github.com/cap-js/ord/blob/debcbd5c86e203b16c01a80336003818fa8660ba/lib/metaData.js#L39-L40)

Reply:
ok, with an old compiler version being installed, I still got the query section. @zongqichen is working on a fix for the dependencies in #163 .